client/src/components/Donation/donation-modal.tsx

```diff
@@ -153,7 +153,8 @@ function DonateModal({
       bsSize='lg'
       className='donation-modal'
       onExited={handleModalHide}
-      show={show}
+      // show={show} linechanged to true
+      show={true}
```

Comment:
This should be done only for local testing. When submitting your pr, please remove this change so the tests evaluate your changes properly.